ServerSide/yoty/Startup.cs

```diff
@@ -62,15 +78,16 @@ public void ConfigureServices(IServiceCollection services)
                     UseRecommendedIsolationLevel = true,
                     DisableGlobalLocks = true
                 }));
-
             JobStorage.Current = new SqlServerStorage(connectionString);
+            services.AddScoped<BidsUpdateJobs>();
             // take this line out of comment when DB exists!
-            // RecurringJob.AddOrUpdate("UpdateBidsDaily",() => BidsUpdateJobs.UpdateBidsPhaseDaily(), Cron.Daily, TimeZoneInfo.Local);
         }
 
         // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
-        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IMapper mapper)
+        public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IMapper mapper , BidsUpdateJobs bids)
         {
+            RecurringJob.AddOrUpdate("UpdateBidsDaily", () => bids.UpdateBidsPhaseDaily(), Cron.MinuteInterval(3), TimeZoneInfo.Local);
```

Comment:
this runs after  
           JobStorage.Current = new SqlServerStorage(connectionString);